src/components/adaptive-cards/Spacer/Spacer.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import webexComponentClasses from '../../helpers';
+
+/**
+ * Spacer helper component
+ *
+ * @param {object} props  React props passed to the component
+ * @param {string} props.spacing  The type of spacing
+ * @param {boolean} props.separator  Whether or not to display the separator
+ * @param {object} props.containerType  The type of the parent container
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
+ * @returns {object} JSX of the component
+ */
+export default function Spacer({
+  className,
+  separator,
+  spacing,
+  style,
+}) {
+  const spacingType = spacing;
```

Comment:
can't we use `spacing` as `separator` as they are, without these additional variables?